docs/pages/includes/diagnostics/debug-service-intro.mdx

```diff
@@ -0,0 +1,4 @@
+Teleport includes an internal service called "Debug". This service contains
```

Comment:
Since "Debug" is a regular English word, this phrasing seems a bit jarring to me. I would say  something like,

"Teleport can run the Debug Service, which contains..."